omnigibson/examples/teleoperation/robot_teleoperate_demo.py

```diff
@@ -0,0 +1,153 @@
+"""
+Example script for using external devices to teleoperate a robot.
+"""
+import omnigibson as og
+from omnigibson.objects import USDObject
+from omnigibson.utils.ui_utils import choose_from_options
+
+ROBOTS = {
+    "FrankaPanda": "Franka Emika Panda (default)",
+    "Fetch": "Mobile robot with one arm",
+    "Tiago": "Mobile robot with two arms",
+}
+
+SYSTEMS = {
+    "SteamVR": "SteamVR with HTC VIVE through OmniverseXR plugin (default)",
+    "Oculus": "Oculus Reader with Quest 2",
+    "Spacemouse": "Spacemouse",
```

Comment:
This is great! Should we also include the keyboard interface here?

Reply:
Since we already have a keyboard interface in `ui_utils.py`, I don't think it's necessary to have another one inherited from TeleopSystem, plus we have a `robot_control_example.py` dedicated for showing keyboard control.

Reply:
Add a think wrapper for the current keyboard controller

Reply:
Turns out the current keyboard controller is not very compatible with the `teleop_utils` logic. I ended up creating a new `KeyboardSystem` class with very similar codes from the `ui_utils`